main.js

```diff
@@ -340,18 +338,26 @@ function _isObject({credential}) {
  * @param {object} options - Options to use.
  * @param {object} options.credential - A VC with a credentialStatus.
  *
- * @returns {Array<object>} An array of statuses.
+ * @returns {Array<object>|null} An array of statuses with type
+ *   "StatusList2021Entry" or null if there are no matching types.
  */
 function _getStatuses({credential}) {
   const {credentialStatus} = credential;
-  if(Array.isArray(credentialStatus)) {
-    return credentialStatus;
+  if(Array.isArray(credentialStatus) && credentialStatus.length > 0) {
+    const statuses = credentialStatus.filter(
+      cs => cs.type === 'StatusList2021Entry');
+    if(statuses.length !== 0) {
+      return statuses;
+    }
+    return null;
+  }
+  if(credentialStatus && credentialStatus.type === 'StatusList2021Entry') {
+    return [credentialStatus];
   }
-  return [credentialStatus];
+  return null;
```

Comment:
Upon further reflection, what does returning `null` instead of an empty array buy us here? Wouldn't it be simpler if the return value of this method was always an array -- and if callers can if it's empty or not they should just check `.length` on that value?

Reply:
Updated to return empty array instead of null here [5c0e9df](https://github.com/digitalbazaar/vc-status-list/pull/6/commits/5c0e9df1cc54c8b1dbb7683c78a19aa5debe54e7)